.eslintrc.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
```

Comment:
We're now using the config files from `eslint.config.js`, this should be removed